### PR TITLE
Add DG and UG

### PR DESCRIPTION
**Pull request by tanshiyu1999**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -239,6 +239,49 @@ Jobby data are saved automatically as a JSON file `[JAR file location]/data/jobb
...
+
+Applying to a Organization by indicating it with the organization's index in the address book or the organization's unique id.
+
+This can be done by supplying the organization's `index` or `id` as the preamble.
+
+Details specifically will be explained the next sections.
+
+
+#### Applying to an Organization: `add org index/org id`
+
```
Comment (wxwern) [inline docs/UserGuide.md:257]: ps: command formats are incorrect; there are a few typos around here; note that `INDEX` and `ORG` should be capitalized by convention and put together with no space in `INDEX/ORG`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -239,6 +239,89 @@ Jobby data are saved automatically as a JSON file `[JAR file location]/data/jobb
...
+* `apply 1 --title SWE`
+
+* `apply id_12345_1 --title Unit Tester --by 12-12-2023`
+
+* `apply id_12345_1 --title Unit Tester --desc Unit testing for Google --by 12-12-2023 --stage resume`
+
+* `apply id_12345_1 --title Junior Engineer --desc Junir role --by 12-12-2023 --stage resume --stat pending`
+
+
+### Deleting a job application
```
Comment (CJ-Lee01) [inline docs/UserGuide.md:283]: This part is really well written!

**Review by CJ-Lee01 — APPROVED** [review]
Looks good to me.

**Review by wamps-jp — APPROVED** [review]
Looks good! The UG section is fine, the DG section will be updated further.

**Pull request merged**